bundles/org.eclipse.swt/Eclipse SWT PI/gtk/library/gtk4.h

```diff
@@ -22,4 +22,12 @@
 
 #include <gtk/gtk.h>
 
+// Hard-link code generated from GTK4.java to LIB_GTK
+#define GTK4_LOAD_FUNCTION(var, name) LOAD_FUNCTION_LIB(var, LIB_GTK, name)
+
+#ifndef GtkFileDialog
```

Comment:
You should not need this define, the correct thing to do is to remove the @param like with cast=(GtkFileDialog *) and it will be dynamically figured (no type cast).

Reply:
done